src/ui/js/Game.js

```diff
@@ -2789,11 +2783,16 @@ Game.isDieClickable = function(isPlayerActive, player, die) {
   }
 };
 
-Game.createDieBorderDiv = function(color) {
-  return $('<div>', {
+Game.createDieBorderDiv = function(color, isSelected) {
+  borderDiv = $('<div>', {
     'class': 'die_border',
-    'style': 'border: 2px solid ' + color,
   });
+
+  if (isSelected) {
```

Comment:
Hmm, isn't this backwards?  Shouldn't you be installing the mat-colored background if the die is *not* selected?